### PR TITLE
Swagger Release 2024-03-01 Suppress Lint Violation

### PR DESCRIPTION
**Pull request by kamboj-prjwl**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

File: specification/network/resource-manager/readme.md [hunk 1]
```diff
@@ -100,6 +100,37 @@ input-file:
   - Microsoft.Network/stable/2024-03-01/vmssNetworkInterface.json
   - Microsoft.Network/stable/2024-03-01/vmssPublicIpAddress.json
   - Microsoft.Network/stable/2024-03-01/webapplicationfirewall.json
+suppressions:
```
Comment (ramoka178) [inline specification/network/resource-manager/readme.md:103]: please add the API path as well, instead of suppressing for the whole file.

**Comment by ramoka178** [comment]
Are suppressions bypassed the last time ? why are only suppressions being added now , without any swagger review ?

**Comment by ramoka178** [comment]
> Are suppressions bypassed the last time ? why are only suppressions being added now , without any swagger review ?

@kamboj-prjwl Could you please answer this for context too ? Would be good to have if this PR is reviewed by a different reviewer next time. 
https://github.com/Azure/azure-rest-api-specs/pull/30372#issue-2495396881 was also not filled. Would be good to know the context of this PR.

**Comment by MUHAMMADIHSANALIKAP** [comment]
Thanks for sharing. Acknowledged and approved.

Regards,
Contact: +1 (448) 219-1552,
Contact: +92 325 465 2843,
Contact: +92 321 389 9048,
MUHAMMAD (MUHAMMAD IHSAN) IHSAN ALI (IHSAN ALI),
Sent From My iPhone 13 Pro Max (Serial No.: Q694WWGX7X).

**Pull request closed without merging**